app/lib/space_routes.rb

```diff
@@ -8,6 +8,7 @@ def self.append_routes(router)
     router.resources :rooms, only: %i[show edit update new create destroy] do
       Furniture.append_routes(router)
       router.resources :furnitures, only: %i[create edit update destroy]
+      router.resources :images, controller: "room/images"
```

Comment:
Slight preference for `router.resource :hero_image`, since that is the resource being managed; not an arbitrary collection of images.